browser-test/src/support/applicant_questions.ts

```diff
@@ -694,4 +694,22 @@ export class ApplicantQuestions {
     await this.submitFromReviewPage()
     await this.page.click('text=End session')
   }
+
+  async expectMayBeEligibileHeadingToBeVisible() {
```

Comment:
**question:** Should this be `expectMayBeEligibleAlertToBeVisible`?

Reply:
Good idea!